board/variscite/imx8mm_var_dart/imx8mm_var_dart.c

```diff
@@ -275,6 +275,9 @@ int board_late_init(void)
 			case 'A':
 				env_set("leica_board", "AP20-PT1");
 				break;
+			case 'D':
```

Comment:
Please add case 'B' and 'C' to make it clear that these belong to "AP20-PT1"

Reply:
```suggestion
			case 'A':
			case 'B':
			case 'C':
			    env_set("leica_board", "AP20-PT1");
			    break;
			case 'D':
			...
```

Reply:
@htho-lgs So A, B and C need to be set as AP20-PT1?

Reply:
yes